content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -116,6 +120,43 @@ The CSV name currently installed must be in the `skips` field of a CSV in the ne
 
 The version currently installed must be in the `olm.skipRange` field of a CSV in the new channel.
 
+
+`replaces and skips example:`
```

Comment:
This renders strangely. I don't think we need to set up an example using brackets, and can just provide a brief description on what the following example is trying to showcase.

Reply:
Yeah just add a header like we have below for `#### Example for SkipRange` if you want to demarcate this.

Reply:
Agree with you. Modified according to suggestion.